lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -16,78 +15,53 @@ describe('NxTextInput', function() {
       },
       getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxTextInput, minimalProps);
 
-  it('renders a text input wrapped in a NxTooltip by default', function() {
-    expect(getShallowComponent()).toMatchSelector(NxTooltip);
+  it('renders a text input', function() {
     expect(getShallowComponent().find('input')).toExist();
     expect(getShallowComponent().find('input')).toHaveProp('type', 'text');
   });
 
-  it('renders a password input wrapped in a tooltip if type is "password"', function() {
-    expect(getShallowComponent({ type: 'password' })).toMatchSelector(NxTooltip);
+  it('renders a password input if type is "password"', function() {
     expect(getShallowComponent({ type: 'password' }).find('input')).toExist();
     expect(getShallowComponent({ type: 'password' }).find('input')).toHaveProp('type', 'password');
   });
 
   it('renders a textarea if type is "textarea"', function() {
-    expect(getShallowComponent({ type: 'textarea' })).toMatchSelector(NxTooltip);
     expect(getShallowComponent({ type: 'textarea' }).find('textarea')).toMatchSelector('textarea');
     expect(getShallowComponent({ type: 'textarea' }).find('textarea')).toHaveProp('type', undefined);
   });
 
   it('sets the value as specified', function() {
-    expect(getShallowComponent({ value: 'foo' }).find('input')).toHaveProp('value', 'foo');
+    expect(getShallowComponent({ value: 'foo' }).find('textarea')).toHaveProp('value', 'foo');
```

Comment:
This shouldn't work

Reply:
> This shouldn't work

I like that your comment leaves it ambiguous about whether it does, or does not in fact work.

Reply:
> I like that your comment leaves it ambiguous about whether it does, or does not in fact work.

Hey it's not my job to run your tests :grin: